### ISSUE TITLE
VTODO (tasks) support

### ISSUE THREAD

**Issue by wetneb** [jazzband/django-ical#59]
The icalendar library, that django-ical relies on, supports not just events but also other types of calendar contents, such as "VTODO" elements (as the [Todo class](https://icalendar.readthedocs.io/en/latest/api.html#icalendar.cal.Todo), sadly not very extensively documented). Those correspond to tasks to be completed by a certain deadline, as defined by the [Tasks extension of the iCalendar specification](https://datatracker.ietf.org/doc/draft-ietf-calext-ical-tasks/).

It seems to me that django-ical does not currently support exposing such tasks in the ical feeds it generates.
Would this functionality be in scope of this library?

For instance, I could imagine using a new method on the feed class to determine whether an entry should be translated as an `Event` or as a `Todo`:
```python
class EventFeed(ICalFeed):

    def item_type(self, item):
        return "todo"
```
Or simply using a class attribute (if calendars should not be able to contain mixes of events and tasks). Other methods would be added to support the fields that are specific to tasks.

**Comment by aleksihakli** [comment]
Seems like a good addition 👍 Do you have a good idea or draft code available for e.g. a PR on this?

**Comment by wetneb** [comment]
I'll have a go at it, I just wanted to check first that it was in scope.

**Comment by aleksihakli** [comment]
Sure sounds like a good idea. I updated the package and Python versions a bit as well, so everything should be ready for PRs :)

**Comment by aleksihakli** [comment]
Released this (implementation from #61) as part of 1.9.0.
